snuba/web/rpc/common/debug_info.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from sentry_protos.snuba.v1.request_common_pb2 import (
+    QueryInfo,
+    QueryMetadata,
+    QueryStats,
+    ResponseMeta,
+    TimingMarks,
+)
+
+from snuba.utils.metrics.timer import Timer
+from snuba.web import QueryResult
+
+
+def extract_response_meta(
+    request_id: str, debug: bool, query_results: List[QueryResult], timers: List[Timer]
```

Comment:
Why are `query_results` and `timers` lists? When would there be more than one of each?

Reply:
Composite query